src/components/drawer.tsx

```diff
@@ -18,7 +18,7 @@ export const Drawer = ({ children, open }: DrawerProps) => {
   return (
     <div
       className={clsx(
-        "fixed top-10 bottom-0 right-0 z-drawer w-96",
+        "fixed top-10 bottom-0 right-0 z-drawer max-[400px]:w-full max-[800px]:w-1/2 w-1/4",
```

Comment:
We can take advantage of [built-in default breakpoints](https://tailwindcss.com/docs/responsive-design) like `sm` and `md` to be consistent across the whole project.

I also recommend keeping the static `w-96` for the bigger screens instead of `w-1/4` that's relative to the screen size. `w-1/4` makes the navbar almost 1000 pixels wide on 4k screens. `w-96` however will always be `384px` wide, which already takes plenty of space.

```suggestion
        "fixed top-10 bottom-0 right-0 z-drawer w-full sm:w-1/2 md:w-96",
```